flight/Modules/Stabilization/stabilization.c

```diff
@@ -944,10 +943,11 @@ static void SettingsUpdatedCb(UAVObjEvent * ev)
 		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this
 		// calculation
 		const float fakeDt = 0.0025f;
+
 		if(settings.GyroTau < 0.0001f)
 			gyro_alpha = 0;   // not trusting this to resolve to 0
 		else
-			gyro_alpha = expf(-fakeDt  / settings.GyroTau);
+			gyro_alpha = (1 / (1 +  ((2.0f * (float)(M_PI) * dT) / settings.GyroTau) ));
```

Comment:
This doesn't seem to be the correct form.  exp(-T / tau) is the correct way to get an IIR alpha for a given time constant.